101-migrate-project-create/azuredeploy.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
format the json, e.g. SHIFT+ALT+F in VS Code

Reply:
Have fixed this. Not sure how to close this